src/pages/Overview/OverviewCardContentExpanded.tsx

```diff
@@ -69,6 +73,15 @@ class OverviewCardContentExpanded extends React.Component<Props> {
         <OverviewCardBars status={this.props.status} />
         <div style={{ marginTop: -20, position: 'relative' }}>
           <Text component={TextVariants.h2} style={{ marginTop: 0 }}>
+            {status.inIdle.length > 0 && (
+              <OverviewStatus
+                id={name + '-undesired'}
```

Comment:
Perhaps better `id={name + '-undesired'}` -> `id={name + '-iddle'}`
Undesired has not meaning on this context.